ansible_collections/arista/avd/roles/eos_designs/python_modules/l3_edge/utils.py

```diff
@@ -0,0 +1,318 @@
+from __future__ import annotations
```

Comment:
Something went wrong in a rebase. Core interfaces and l3edge have been collapsed to a single new module.